packages/pds/src/app-view/services/feed/index.ts

```diff
@@ -341,19 +353,93 @@ export class FeedService {
       this.getPostInfos(Array.from(postUris), requester),
       this.services.label.getLabelsForSubjects([...postUris, ...actorDids]),
     ])
-    const embeds = await this.embedsForPosts(posts, requester)
+    const blocks = await this.blocksForPosts(posts)
+    const embeds = await this.embedsForPosts(posts, blocks, requester)
 
     return this.views.formatFeed(
       items,
       actors,
       posts,
       embeds,
       labels,
+      blocks,
       usePostViewUnion,
     )
   }
 
-  async embedsForPosts(postInfos: PostInfoMap, requester: string, depth = 0) {
+  // applies blocks for visibility to third-parties (i.e. based on post content)
+  async blocksForPosts(posts: PostInfoMap): Promise<PostBlockState> {
```

Comment:
this fn is a little bit hard to follow altho the logic all seems sound

I think two things might clear it up
- finding a new word for "potential block" other than "block" - it's a bit overloaded & I have to look for context to see if a "block" is a potential or actual
- using "parent" instead of reply. I kinda wish our lexicons reflected this as well - but the thing that is "blocking" a given post is it's "parent" not it's "reply". put another way: it's the post in question is not "blocking a reply" it's "blocked in the context of being a reply"